bleep-cli/src/scala/bleep/commands/BuildUpdateDeps.scala

```diff
@@ -32,6 +39,25 @@ case class BuildUpdateDeps(scalaStewardMode: Boolean, allowPrerelease: Boolean)
       Await.result(fetchAllVersions(fileCache, repos, allDeps), Duration.Inf)
     }
 
+    singleDepParsed match {
```

Comment:
can you extract all of the logic here into an `object` and write some tests for it?